instana/instrumentation/sqlalchemy.py

```diff
@@ -47,8 +49,18 @@ def receive_after_cursor_execute(**kw):
             if scope is not None:
                 scope.close()
 
-    @event.listens_for(Engine, 'dbapi_error', named=True)
-    def receive_dbapi_error(**kw):
+
+    # Deprecated since version 0.9: The ConnectionEvents.dbapi_error() event is
+    # deprecated and will be removed in a future release. Please refer to the
+    # ConnectionEvents.handle_error() event.
+    if sqlalchemy.__version__[0] != "0":
+        error_event = "handle_error"
+    else:
+        error_event = "dbapi_error"
```

Comment:
A minor note: I believe it's safer (and slightly more readable) to inititalize the value first, and update it if needed:
```python
error_event = "handle_error"

# Handle 0.x explicitly
if sqlalchemy.__version__[0] == "0":
  error_event = "dbapi_error"
```